src/router/index.ts

```diff
@@ -120,6 +120,11 @@ if (domain) {
     },
     { path: '/timeline', name: 'timeline', component: TimelineView },
     { path: '/ranking', name: 'ranking', component: RankingView },
+    {
+      path: '/payment/network',
+      name: 'paymentNetworkFees',
+      component: PaymentNetworkFees
```

Comment:
```suggestion
      path: '/payment/:type',
      name: 'payment',
      component: PaymentView
```
Would be cleaner to use sub-routes like we do with the `SpaceView`. Then we have `PaymentView` as base view and you can create another view called `PaymentNetwork`

Reply:
How will url looks like ? `/payment?type='network` ? What will just `/payment` redirect to ?

Reply:
The `path` for `PaymentView` will be `/payment` and the path for `PaymentNetwork` will be `network`. The complete path will be just like you have now `/payment/network`. `/payment` will redirect to nothing for now

Reply:
PaymentNetwork component is in `children` of PaymentView